02-peer-protocol.md

```diff
@@ -841,7 +841,9 @@ dust exposure. The exact value used is a matter of node policy.
 
 For channels that don't use `option_anchors_zero_fee_htlc_tx`, an increase of
 the `feerate_per_kw` may trim multiple htlcs from commitment transactions,
-which could create a large increase in dust exposure.
+which could create a large increase in dust exposure. If `option_anchors_zero_fee_htlc_tx`
+has been negotiated, the HTLC feerate is implied to be 0, the checks should
+still be applied as a node can be exposed due to the sum of the trimmed HTLCs.
```

Comment:
This sentence is quite unclear to me: which checks are you talking about? I don't think this new sentence is necessary at all to be honest, the requirements above already apply to all commitment types, and this paragraph only says that without `option_anchors_zero_fee_htlc_tx` you should be extra cautious when the feerate changes.

Reply:
Removed this sentence on latest commit `7705356`, if we think it doesn't add clarification.